source/OutfitterPanel.cpp

```diff
@@ -178,8 +182,15 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point)
 			minCount = maxCount = player.HasMapped(mapSize);
 		else
 		{
+			highlightDifferences = true;
+			string lastModelName;
```

Comment:
`lastModelName` suggests this is the model name of the last ship that was processed.
This doesn't appear to match its usage, where it is set once (with the model name of the first ship) and then never updated again.
`firstModelName` seems like it better matches how the variable is actually used.